app/scripts/controllers/eventDashboard.js

```diff
@@ -41,10 +41,6 @@ angular
             templateUrl: createEventModalTemplate,
           })
           .result.then(function (conferenceName) {
-            if (!conferenceName) {
-              return;
-            }
-
```

Comment:
Are you removing this since the conference name is always required?

Reply:
Yes